zingolib/src/error.rs

```diff
@@ -1,3 +1,5 @@
+#[derive(Debug)]
 pub enum ZingoLibError {
     NoWalletLocation,
+    AbsurdTransactionMetadata,
```

Comment:
"Absurd" in what way?

Reply:
if we take the metadata to be true, the transaction created a whole bunch of z from thin air